src/tabs/scss/tabs.scss

```diff
@@ -0,0 +1,50 @@
+@import "inc/bootstrap";
+
+/** Styles for a group of buttons to be glued together into a tabs-like bar
+ *  Intended use:
+ *  <ul class="tab-group">
+ *    <li class="tab">
+ *      <button>
+ */
+.tab-group {
+    list-style: none;
+    @include simple-border($websiteBorder);
+    @include border-radius(4);
+    padding: 1px;
+    display: inline-flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+
+    .tab {
+        button {
+            @include border-radius(0);
+            padding: 0 15px;
+            background-color: white;
+            color: $textColor !important; // override button text color from basic.scss
```

Comment:
Maybe important is not needed (would be better to avoid it)

Reply:
You are right.